React/Views/RCTViewManager.m

```diff
@@ -24,6 +24,58 @@
 #import "RCTTVView.h"
 #endif
 
+#if TARGET_OS_OSX
+@implementation RCTConvert (UIView)
+
+RCT_ENUM_CONVERTER(
+    RCTCursor,
+    (@{
+      // CSS cursor names
+      @"auto" : @(RCTCursorAuto),
+      @"default" : @(RCTCursorArrow),
+      @"contextMenu" : @(RCTCursorContextualMenu),
+      @"pointer" : @(RCTCursorPointingHand),
+      @"text" : @(RCTCursorIBeam),
+      @"verticalText" : @(RCTCursorIBeamCursorForVerticalLayout),
+      @"alias" : @(RCTCursorDragLink),
+      @"copy" : @(RCTCursorDragCopy),
+      @"noDrop" : @(RCTCursorOperationNotAllowed),
+      @"notAllowed" : @(RCTCursorOperationNotAllowed),
+      @"grab" : @(RCTCursorOpenHand),
+      @"grabbing" : @(RCTCursorClosedHand),
+      @"colResize" : @(RCTCursorResizeLeftRight),
+      @"rowResize" : @(RCTCursorResizeUpDown),
+      @"nResize" : @(RCTCursorResizeUp),
+      @"eResize" : @(RCTCursorResizeRight),
+      @"sResize" : @(RCTCursorResizeDown),
+      @"wResize" : @(RCTCursorResizeLeft),
+
+      // macOS cursor names
+      @"arrow" : @(RCTCursorArrow),
+      @"iBeam" : @(RCTCursorIBeam),
+      @"crosshair" : @(RCTCursorCrosshair),
+      @"closedHand" : @(RCTCursorClosedHand),
+      @"openHand" : @(RCTCursorOpenHand),
+      @"pointingHand" : @(RCTCursorPointingHand),
+      @"resizeLeft" : @(RCTCursorResizeLeft),
+      @"resizeRight" : @(RCTCursorResizeRight),
+      @"resizeLeftRight" : @(RCTCursorResizeLeftRight),
+      @"resizeUp" : @(RCTCursorResizeUp),
+      @"resizeDown" : @(RCTCursorResizeDown),
+      @"resizeUpDown" : @(RCTCursorResizeUpDown),
+      @"disappearingItem" : @(RCTCursorDisappearingItem),
+      @"iBeamCursorForVerticalLayout" : @(RCTCursorIBeamCursorForVerticalLayout),
+      @"operationNotAllowed" : @(RCTCursorOperationNotAllowed),
+      @"dragLink" : @(RCTCursorDragLink),
+      @"dragCopy" : @(RCTCursorDragCopy),
+      @"contextualMenu" : @(RCTCursorContextualMenu),
```

Comment:
Can we stick to the official CSS snake-case names? https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

Reply:
Makes sense to me. I'm including "disappearing-item" as well which as far as I can tell has no equivalent in CSS.